packages/zapp/console/tsconfig.json

```diff
@@ -12,5 +12,9 @@
         "noImplicitOverride": false
     },
     "references": [{ "path": "../../plugins/components" }],
-    "include": ["src/**/*"]
+    "include": [
+        "src/**/*",
+        // TODO: *.json could be removed when tsconfig.build.json would be properly consumed by webpack
+        "src/**/*.json"
```

Comment:
just wonder isn't 
`"src/**/*` include `"src/**/*.json`

Reply:
🙃 I know it's sound funky, but no `**/*` doesn't include `*.json` files (and some other file types too). It was specifically done by TS - to avoid accidental addition of big files, which are rarely should be part of the project.
As webpack will currently include test/spec/ mock files, we would need to build in few jsons that our tests are relied upon. (for now)